README.md

```diff
@@ -155,28 +155,46 @@ The `decrypt` function returns a string with the plaintext data.
 ### Lock context
 
 `jseql` supports lock contexts to ensure that only the intended users can access sensitive data.
+To use a lock context, initialize a `LockContext` object with the identity claims and workspace ID.
 
-To use a lock context, you will need to provide an array of identity claims.
+```typescript
+import { LockContext } from '@cipherstash/jseql'
+
+const lc = new LockContext({
+  identityClaim: ['sub'],
+  workspaceId: 'your-workspace-id',
+  region: 'ap-southeast-2', // optional, defaults to ap-southeast-2
```

Comment:
What's the reason for putting `workspaceId` and `region` on the lock context instead of the client?

Reply:
Yeah that's a good call - I'll update